modeshape-jcr/src/main/java/org/modeshape/jcr/NodeTypes.java

```diff
@@ -238,6 +240,10 @@ protected NodeTypes( ExecutionContext context,
                     nodeTypeNamesThatAreShareable.add(name);
                 }
 
+                if (nodeType.isNodeType(ModeShapeLexicon.UNORDERED_COLLECTION)) {
+                    nodeTypeNamesThatAreUnorderableCollections.add(name);
```

Comment:
How are the tiny, small, large, and huge unordered collection node types added to the `nodeTypeNamesThatAreUnorderableCollections `?